src/trixi.f90

```diff
@@ -2,22 +2,22 @@ module LibTrixi
   implicit none
 
   interface
-    subroutine trixi_initialize(project_directory) bind(c)
+    subroutine trixi_initialize_c(project_directory) bind(c, name='trixi_initialize')
```

Comment:
Just to get it right: calling `trixi_initialize_c` will now internally call libtrixi's `trixi_initalize`?

Reply:
Yes. The entity provided by `name='...'` is the name of the C function.